test/integration/render-tests/line-dasharray/function--line-width-composite-function/style.json

```diff
@@ -0,0 +1,150 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "height": 256,
+      "ignored":{
+        "native": "https://github.com/mapbox/mapbox-gl-js/issues/3682#issuecomment-264348200"
+      }
+    }
+  },
+  "center": [
+    13.418056,
+    52.499167
+  ],
+  "zoom": 14,
+  "sources": {
+    "mapbox": {
+      "type": "vector",
+      "maxzoom": 14,
+      "tiles": [
+        "local://tiles/{z}-{x}-{y}.mvt"
+      ]
```

Comment:
I'd be in favor of simplifying these render tests using some bare-bones GeoJSON, as that'll make em easier to understand/assess.